user_story/rent_a_room.md

```diff
@@ -0,0 +1,29 @@
+User Story (for rent)
+====
+
+
+Name:
+----
+birana
+
+
+Description:
+---- 
+As a student who looking for rent in the coming term, I want to search for a room through the website. So that I can access to plent of rooms for rent and choose my favorite. 
```

Comment:
"plenty of rooms"？

Reply:
well,

Reply:
well, I mean the resource of rooms is rich and plentiful ... dose it make sense?

Reply:
Yes it makes sense but in the file you wrote "So that I can access to plent of rooms", missing a "y" in the word "plent"?

Reply:
〒▽〒 thanks. I already correct it